src/main/java/com/sngular/kloadgen/processor/util/SchemaProcessorUtils.java

```diff
@@ -295,7 +326,7 @@ private static void extracted(
         msgDef.addField("repeated", "typemapnumber" + elementField.getName(), elementField.getName(), elementField.getTag());
 
         msgDef.addMessageDefinition(
-            MessageDefinition.newBuilder("typemapnumber" + elementField.getName()).addField(OPTIONAL, "string", "key", 1).addField(OPTIONAL, realType, "value", 2).build());
+          MessageDefinition.newBuilder("typemapnumber" + elementField.getName()).addField(OPTIONAL, "string", "key", 1).addField(OPTIONAL, realType, "value", 2).build());
```

Comment:
You use the value of "typemapnumber" several times, so it would be nice to create a variable with that value.